package/Dockerfile

```diff
@@ -1,25 +1,39 @@
-FROM registry.suse.com/bci/bci-base:15.5
+ARG BCI_VERSION=15.6
 
-ENV SSL_CERT_DIR /etc/rancher/ssl
+FROM registry.suse.com/bci/bci-busybox:${BCI_VERSION} AS final
+FROM registry.suse.com/bci/bci-base:${BCI_VERSION} AS builder
+
+# Creates the base dir for the target image, and hydrates it with the
+# final image's contents.
+RUN mkdir /chroot
+COPY --from=final / /chroot/
 
-RUN zypper -n update && \
-    zypper -n install git-core curl ca-certificates unzip mkisofs xz gzip sed tar openssh-clients && \
+RUN zypper --non-interactive refresh && \
+    zypper --installroot /chroot -n rm busybox-less && \
+    zypper --installroot /chroot -n install \
+        git-core curl mkisofs openssh-clients && \
     zypper -n clean -a && \
-    rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/*
+    rm -rf /chroot/tmp/* /chroot/var/tmp/* /chroot/usr/share/doc/packages/*
 
 RUN useradd -u 1000 machine
+RUN cp /etc/passwd /chroot/etc/passwd
+
+COPY download_driver.sh /chroot/usr/local/bin/
+RUN chmod +x /chroot/usr/local/bin/download_driver.sh
+
+COPY rancher-machine entrypoint.sh /chroot/usr/local/bin/
+RUN chmod 0777 /chroot/usr/local/bin
```

Comment:
Should we take this opportunity to remove the permissive global write permissions?